src/Apps/Order/Routes/__tests__/Shipping.test.tsx

```diff
@@ -84,12 +89,17 @@ describe("Shipping", () => {
   })
 
   describe("mutation", () => {
+    beforeEach(() => {
+      console.error = jest.fn() // Silences component logging.
+      commitMutation.mockReset()
+    })
+
     it("routes to payment screen after mutation completes", () => {
       const component = getWrapper(testProps)
       const mockCommitMutation = commitMutation as jest.Mock<any>
       mockCommitMutation.mockImplementationOnce(
         (_environment, { onCompleted }) => {
-          onCompleted()
+          onCompleted(settingOrderShipmentSuccess)
```

Comment:
@ds300, could `MockRelayRenderer` be used in place of mocking `createFragmentContainer`? I was thinking that it would be useful for these cases, but I'm not 100% certain.